zookeeper-server/src/main/java/org/apache/zookeeper/JLineZNodeCompleter.java

```diff
@@ -43,40 +45,30 @@ public int complete(String buffer, int cursor, List candidates) {
         }
 
         if (token.startsWith("/")) {
-            return completeZNode(buffer, token, candidates);
-        }
-        return completeCommand(buffer, token, candidates);
-    }
-
```

Comment:
We can still use this skeleton to make the code more refined and less code changes?

Reply:
Yes! I just focus on the function but the lines is a little more. I fixed it in the commit https://github.com/apache/zookeeper/pull/1458/commits/16010357021235e9e32b21fe38b3ba2192490630